storage-xray-udp/src/main/java/zipkin2/storage/xray_udp/XRayFormatter.java

```diff
@@ -0,0 +1,64 @@
+/**
+ * Copyright 2016-2017 The OpenZipkin Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package zipkin2.storage.xray_udp;
+
+import com.squareup.moshi.JsonWriter;
+import java.io.IOException;
+import okio.Buffer;
+
+/**
+ * This class is an utility class providing some basic ways to create simple JSON's.
+ * Since a span can be annotated by only a String value, passing input as JSON is required.
+ */
+public final class XRayFormatter {
+
+  /**
+   *
+   * Formating an exception to be consumed by XRay by Spans annotations
+   *
+   * @param exceptionId an unique exception id @see brave.internal.Platform#randomLong()
+   * @param isRemote Any calls to a remote service/broker should be true
+   * @param maxStackTraceElement XRay has a limit of 60KB per segment. Choose a wise number
```

Comment:
This would be global configurable as ex transports have limits that can be much smaller

Reply:
How can I configure this number since the method is static? Should I declare another static variable can be changeable? Or should I make it static class and force users to create an instance of it?